projects/plugins/wpcomsh/widgets/tlkio/tlkio.js

```diff
@@ -1,28 +1,26 @@
 ( function ( window, document ) {
-
-	var tlkioOnload = function ( e ) {
-		var target_elements  = document.querySelectorAll( '.tlkio-container' );
-		for ( var i = 0, len = target_elements.length; i < len; ++i ) {
-			var channel_name    = target_elements[i].getAttribute( 'data-channel' ),
-			custom_css_path     = target_elements[i].getAttribute( 'data-theme' ),
-			nickname            = target_elements[i].getAttribute( 'data-nickname' ),
-			iframe              = document.createElement( 'iframe' );
+	const tlkioOnload = function () {
+		const target_elements = document.querySelectorAll( '.tlkio-container' );
+		for ( let i = 0, len = target_elements.length; i < len; ++i ) {
+			const channel_name = target_elements[ i ].getAttribute( 'data-channel' ),
+				custom_css_path = target_elements[ i ].getAttribute( 'data-theme' ),
+				nickname = target_elements[ i ].getAttribute( 'data-nickname' ),
+				iframe = document.createElement( 'iframe' );
 
 			// var iframe_src = 'http://embed.lvh.me:3000/' + channel_name,
-			var iframe_src = '//embed.tlk.io/' + channel_name,
-			iframe_query = [];
-
+			let iframe_src = '//embed.tlk.io/' + channel_name;
+			const iframe_query = [];
 
 			if ( custom_css_path && custom_css_path.length > 0 ) {
 				iframe_query.push( 'custom_css_path=' + custom_css_path );
 			}
 
 			if ( nickname && nickname.length > 0 ) {
-				iframe_query.push('nickname=' + nickname );
+				iframe_query.push( 'nickname=' + nickname );
 			}
 
 			if ( iframe_query.length > 0 ) {
-				iframe_src += '?' + iframe_query.join ( '&' );
+				iframe_src += '?' + iframe_query.join( '&' );
 			}
 
 			iframe.setAttribute( 'src', iframe_src );
```

Comment:
I see CodeQL is complaining about this line. I suspect what it wants is passing `channel_name`, `custom_css_path`, and `nickname` through `encodeURIComponent()` where they're being appended to the URL above?